### ISSUE TITLE
mautrix-telegram fails to start on nixos-21.05

### ISSUE THREAD

**Issue by pacien** [NixOS/nixpkgs#135583]
The mautrix-telegram service fails to start with the package from nixos-21.05.
The package from unstable/master with the same version number works fine however.

```
Aug 24 20:30:07 host systemd[1]: Started Mautrix-Telegram, a Matrix-Telegram hybrid puppeting/relaybot bridge..
Aug 24 20:30:12 host mautrix-telegram[3935072]: Traceback (most recent call last):
Aug 24 20:30:12 host mautrix-telegram[3935072]:   File "/nix/store/dplwj78rxqikzv707mrv6prppzrl0bsz-python3.8-mautrix-telegram-0.10.1/bin/.mautrix-telegram-wrapped", line 6, in <module>
Aug 24 20:30:12 host mautrix-telegram[3935072]:     from mautrix_telegram.__main__ import main
Aug 24 20:30:12 host mautrix-telegram[3935072]:   File "/nix/store/dplwj78rxqikzv707mrv6prppzrl0bsz-python3.8-mautrix-telegram-0.10.1/lib/python3.8/site-packages/mautrix_telegram/__main__.py", line 25, in <module>
Aug 24 20:30:12 host mautrix-telegram[3935072]:     from .web.provisioning import ProvisioningAPI
Aug 24 20:30:12 host mautrix-telegram[3935072]:   File "/nix/store/dplwj78rxqikzv707mrv6prppzrl0bsz-python3.8-mautrix-telegram-0.10.1/lib/python3.8/site-packages/mautrix_telegram/web/__init__.py", line 1, in <module>
Aug 24 20:30:12 host mautrix-telegram[3935072]:     from .provisioning import ProvisioningAPI
Aug 24 20:30:12 host mautrix-telegram[3935072]:   File "/nix/store/dplwj78rxqikzv707mrv6prppzrl0bsz-python3.8-mautrix-telegram-0.10.1/lib/python3.8/site-packages/mautrix_telegram/web/provisioning/__init__.py", line 31, in <module>
Aug 24 20:30:12 host mautrix-telegram[3935072]:     from ...user import User
Aug 24 20:30:12 host mautrix-telegram[3935072]:   File "/nix/store/dplwj78rxqikzv707mrv6prppzrl0bsz-python3.8-mautrix-telegram-0.10.1/lib/python3.8/site-packages/mautrix_telegram/user.py", line 42, in <module>
Aug 24 20:30:12 host mautrix-telegram[3935072]:     from .abstract_user import AbstractUser
Aug 24 20:30:12 host mautrix-telegram[3935072]:   File "/nix/store/dplwj78rxqikzv707mrv6prppzrl0bsz-python3.8-mautrix-telegram-0.10.1/lib/python3.8/site-packages/mautrix_telegram/abstract_user.py", line 27, in <module>
Aug 24 20:30:12 host mautrix-telegram[3935072]:     from telethon.tl.types import (
Aug 24 20:30:12 host mautrix-telegram[3935072]: ImportError: cannot import name 'UpdatePinnedMessages' from 'telethon.tl.types' (/nix/store/7wrzvhq2i9bk1yrqd6ypykph9rkya35x-python3.8-telethon-1.17.5/lib/python3.8/site-packages/telethon/tl/types/__init__.py)
Aug 24 20:30:12 host systemd[1]: mautrix-telegram.service: Main process exited, code=exited, status=1/FAILURE
Aug 24 20:30:12 host systemd[1]: mautrix-telegram.service: Failed with result 'exit-code'.
```

I believe this follows this backport PR: https://github.com/NixOS/nixpkgs/pull/134933 (CC @Ma27)

**Comment by Ma27** [comment]
@pacien First of all, I have to apologize for this, this shouldn't have happened! It seems as if I deployed the update from a branch (21.05-based) branch where I already have a newer telethon. Considering that 0.9 isn't forward-compatible, I guess the easiest workaround is to update telethon for mautrix-telegram only, correct?

**Comment by pacien** [comment]
Fixed by #135591.
